docs/how-to-guides/develop-a-flow/develop-evaluation-flow.md

```diff
@@ -0,0 +1,167 @@
+
+## Evaluation flow example
+
+In this guide, we use [eval-classification-accuracy](https://github.com/microsoft/promptflow/tree/main/examples/flows/evaluation/eval-classification-accuracy) flow as an example of the evaluation flow. This is a flow illustrating how to evaluate the performance of a classification flow. It involves comparing each prediction to the groundtruth and assigns a `Correct` or `Incorrect` grade, and aggregating the results to produce metrics such as `accuracy`, which reflects how good the system is at classifying the data.
+
+## Flow inputs
+
+The flow `eval-classification-accuracy` contains two inputs:
+
+```yaml
+inputs:
+  groundtruth:
+    type: string
+    description: Groundtruth of the original question, it's the correct label that you hope your standard flow could predict.
+    default: APP
+  prediction:
+    type: string
+    description: The actual predicted outputs that your flow produces.
+    default: APP
+```
+
+As evident from the inputs description, the evaluation flow requires two specific inputs: 
+- `groundtruth`: This input represents the actual or expected values against which the performance of the standard/chat flow will be evaluated.
+- `prediction`: The prediction input is derived from the outputs of another standard/chat flow. It contains the predicted values generated by the standard/chat flow, which will be compared to the groundtruth values during the evaluation process.
+
+From the definition perspective, there is no difference compared with adding an input/output in a `standard/chat flow`. However when running an evaluation flow, you may need to specify the data source from both data file and flow run outputs. For more details please refer to [Run and evaluate a flow](../run-and-evaluate-a-flow/run-and-evaluate-a-flow.md#evaluate-your-flow).
+
+
+## Aggregation node
+
+
+Before introducing the aggregation node, let's see what a regular node looks like, we use node `grade` in the example flow for instance:
+
+```yaml
+- name: grade
+  type: python
+  source:
+    type: code
+    path: grade.py
+  inputs:
+    groundtruth: ${inputs.groundtruth}
+    prediction: ${inputs.prediction}
+```
+
+It takes both `groundtruth` and `prediction` from the flow inputs, compare them in the source code to see if they match:
+
+```python
+from promptflow import tool
+
+@tool
+def grade(groundtruth: str, prediction: str):
+    return "Correct" if groundtruth.lower() == prediction.lower() else "Incorrect"
+```
+
+When it comes to an `aggregation node`, there are two key distinctions that set it apart from a regular node:
+1. It has an attribute `aggregation` set to be `true`.
+
+```yaml
+- name: calculate_accuracy
+  type: python
+  source:
+    type: code
+    path: calculate_accuracy.py
+  inputs:
+    grades: ${grade.output}
+  aggregation: true
```

Comment:
may add a comment here to highlight this line

Reply:
Updated